tests/acceptance/settings-profile-test.js

```diff
@@ -99,12 +100,12 @@ test("it allows editing of users image", (assert) => {
 
   andThen(() => {
     fillInFileInput('.image-drop input', { name: fileName, content: droppedImageString });
```

Comment:
Not sure. Same on other comment above. Maybe @begedin could weigh in? Or @green-arrow potentially?